google-chart.html

```diff
@@ -384,6 +404,7 @@
             var chart = chartAndData[0];
             var data = chartAndData[1];
             google.visualization.events.addOneTimeListener(chart, 'ready', function() {
+              this._setImageUri(this.imageURI);
```

Comment:
I thought `value` only set the default (initial) value.
However, your comment makes me wonder if setting it up as a `computed` property would work.

If a computed property changes in the woods and no-one is observing, does it get computed?